cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,46 @@
-describe('', () => {
+describe('CRUD article', () => {
+  let user;
+  let articleData;
   before(() => {
+    cy.visit(' ');
```

Comment:
You should avoid visiting an empty URL. Please replace the empty string with the URL of the page you want to visit.